test/groovy/com/sap/piper/ConfigurationMergerTest.groovy

```diff
@@ -63,6 +63,17 @@ class ConfigurationMergerTest {
         Assert.assertEquals(null, merged.veggie)
     }
 
+    @Test
+    void testMergeDeepStructureWithMissingDefaults(){
+        Map defaults = [others:[apples: 18]]
+        Map configuration = [fruits: [bananaaas: 50, cucumbers: 1000]]
```

Comment:
Is the spelling of `bananaaas` intended to be like that?

Reply:
It is consistent with the other tests